app/_src/kubernetes-ingress-controller/guides/custom-class/internal-external.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Using Custom Classes to split Internal/External traffic
+---
+
+{{ site.kic_product_name }} automatically creates a `kong` `IngressClass` when installed. All of the example ingress definitions in the documentation set `spec.ingressClassName: kong`, which allows things to work by default.
+
+Advanced users of {{ site.kic_product_name }} may wish to split traffic in to internal and external ingress definitions. This requires multiple {{ site.kic_product_name }} instances, with each pointing to a different `IngressClass`
+
+## Understanding IngressClass
+
+The `IngressClass` resource is what binds an `Ingress` definition to an ingress controller. The value in the `spec.controller` field defines which ingress controller will process those ingress definitions. {{ site.kic_product_name }} processes any `IngressClass` where `spec.controller` is set to `ingress-controllers.konghq.com/kong`.
+
+## Installing Kong
+
+{{ site.kic_product_name }} processes one `IngressClass` per installation. {{ site.kic_product_name }} requires two deployments to split internal and external traffic.
+
+Each deployment lives in it's own namespace, and the `controller.ingressController.ingressClass` value is set depending on if that deployment should handle internal or external traffic.
```

Comment:
```suggestion
Each deployment lives in its namespace, and the `controller.ingressController.ingressClass` value is set depending on whether that deployment should handle internal or external traffic.
```

Reply:
This changes the meaning. It should be `it's own` or words that indicate exclusivity